jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8521,6 +8524,14 @@ public String translateEscapes() {
 					}
 					builder.append((char)octal);
 					break;
+				case '\r':
+					/* Check if the next character is the newline character, i.e. case "\r\n" */
+					if (((index + 1) < strLength) && ('\n' == charArray[index + 1])) {
+						index++;
+					}
+					break;
+				case '\n':
```

Comment:
maybe add a comment for why ignoring `\n` here?

Reply:
added comment to explain line terminator cases